filament/backend/include/private/backend/Program.h

```diff
@@ -31,8 +33,8 @@ class Program {
 public:
 
     static constexpr size_t SHADER_TYPE_COUNT = 2;
-    static constexpr size_t UNIFORM_BINDING_COUNT = 6; //BindingPoints::COUNT;
-    static constexpr size_t SAMPLER_BINDING_COUNT = 6; //BindingPoints::COUNT;
+    static constexpr size_t UNIFORM_BINDING_COUNT = filament::CONFIG_UNIFORM_BINDING_COUNT;
```

Comment:
backend  cannot depend on  filabridge

Reply:
Ok, I will try using DriverEnums instead of EngineEnums

Reply:
Done.